.github/workflows/push_to_aws_ec2.yml

```diff
@@ -0,0 +1,25 @@
+name: Push-to-EC2
+
+on: push
+
+jobs:
+  deploy:
+    name: Push to EC2 Instance
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v1
+
+      - name: Deploy to EC2 instance
+        uses: easingthemes/ssh-deploy@v2.1.5
+        env:
+          ARGS: 
```

Comment:
It's important to note that we may need to add more folders/files here as we continue to grow this project.